uncontrast_studies/models/consciousness_measure.py

```diff
@@ -0,0 +1,74 @@
+from django.db import models
+from django.db.models import CASCADE
+
+
+class UnConsciousnessMeasurePhase(models.Model):
+    name = models.CharField(blank=False, null=False, max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class UnConsciousnessMeasureType(models.Model):
+    name = models.CharField(blank=False, null=False, max_length=30)
+
+    def __str__(self):
+        return self.name
+
+
+class UnConsciousnessMeasureSubType(models.Model):
+    name = models.CharField(blank=False, null=False, max_length=30)
+    type = models.ForeignKey(
+        blank=False,
+        null=False,
+        on_delete=CASCADE,
+        to=UnConsciousnessMeasureType,
+        related_name="unconsciousness_measure_sub_types",
+    )
+
+    def __str__(self):
+        return self.name
+
+
+class UnConsciousnessMeasure(models.Model):
+    experiment = models.ForeignKey(
+        to="uncontrast_studies.UnConExperiment", on_delete=CASCADE, related_name="unconsciousness_measures"
+    )
+    phase = models.ForeignKey(
+        blank=False,
+        null=False,
+        on_delete=CASCADE,
```

Comment:
I think all those three are "PROTECT" you don't want to allow removing phase, type, subtype if it's in use (unlike the experiment which makes sense)